commons/logging/src/main/java/org/operaton/commons/logging/BaseLogger.java

```diff
@@ -276,4 +281,25 @@ protected String exceptionMessage(String id, String messageTemplate, Object... p
     }
   }
 
+  /**
+   *  Sanitize parameters to avoid injection attacks
+   */
+  private Object[] sanitizeParameters(Object... parameters) {
+    if (parameters == null) {
+      return new Object[0];
+    }
+    Object[] sanitized = new Object[parameters.length];
+    for (int i = 0; i < parameters.length; i++) {
+      sanitized[i] = sanitize(parameters[i]);
+    }
+    return sanitized;
+  }
+
+  private Object sanitize(Object parameter) {
+    if (parameter instanceof String) {
```

Comment:
In new code we should use variable reassignement in instanceof itself instead of casting.

Reply:
done